sql/src/main/java/io/crate/analyze/relations/AliasedAnalyzedRelation.java

```diff
@@ -160,13 +160,43 @@ public boolean isDistinct() {
 
     @Override
     public String toString() {
-        return "AliasedAnalyzedRelation{" + "qualifiedName=" + qualifiedName +
-               ", relation=" + relation +
-               '}';
+        return relation + " AS " + qualifiedName;
     }
 
     @Override
     public <C, R> R accept(AnalyzedRelationVisitor<C, R> visitor, C context) {
         return visitor.visitAliasedAnalyzedRelation(this, context);
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
I'm curious why this is needed?

Reply:
Ah right, it shouldn't be needed actually. I'll remove it.